src/lib/draw/AttributeMode.svelte

```diff
@@ -10,11 +10,12 @@
     openFromSidebar,
     formOpen,
   } from "../../stores";
+  import type { Writable } from "svelte/store";
 
   const thisMode = "edit-attribute";
 
   // TODO Just tell us if we're the current mode or not, actually...
-  export let mode: Mode;
+  export let currentMode: Writable<Mode>;
```

Comment:
We don't need to plumb this as a prop. Stores are effectively app-global. We can import this from `stores.ts` like we do other stuff